modules/drivers/redshift/test/metabase/driver/redshift_test.clj

```diff
@@ -60,3 +60,19 @@
                       :nested? false
                       :query-hash (byte-array [-53, -125, -44, -10, -18, -36, 37, 14, -37, 15, 44, 22, -8, -39, -94, 30, 93, 66, -13, 34, -52, -20, -31, 73, 76, -114, -13, -42, 52, 88, 31, -30])})))
           "if I run a Redshift query, does it get a remark added to it?")))))
+
+(deftest parameters-test
+  (testing "Native query parameters should work with filters."
+    (is (= [[693 "2015-12-29T00:00:00Z" 10 90]]
+           (mt/rows
+             (qp/process-query
+              {:database   (mt/id)
```

Comment:
oh hey I just came across needing a good example of a native query like this, that helps a lot

Reply:
There's also https://github.com/metabase/metabase/blob/master/test/metabase/query_processor_test/native_test.clj which is probably the most basic e2e example

Reply:
I just noticed this -- doesn't look like this is testing against redshift